pkg/proc/stack.go

```diff
@@ -387,12 +397,120 @@ func (it *stackIterator) switchStack() bool {
 	}
 }
 
+func (it *stackIterator) switchSp() bool {
```

Comment:
The return value isn't used so it should be removed. Also the name should have a hint that this is arm64 specific (arm64SwitchSp, for example).

Reply:
The return value is left for only change SP and ignore other regs. Need to remove?